packages/web/src/services/audius-sdk/audiusSdk.ts

```diff
@@ -49,7 +51,12 @@ const initSdk = async () => {
     services: {
       discoveryNodeSelector,
       solanaRelay,
-      auth: sdkAuthAdapter
+      audiusWalletClient,
+      ethWalletClient: createWalletClient({
+        account: '0x0', // dummy replaced by relay
+        chain: mainnet,
+        transport: http(`${env.IDENTITY_SERVICE}/ethereum/rpc`)
```

Comment:
nit: We should consider a slightly more robust url-building mechanism for stuff like this. It occurs to me that a trailing slash would break this.